review_practice/react-app/src/components/PriceRangeSlider/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import * as Style from './style';
+
+import Slider from '@material-ui/core/Slider';
+
+const PriceRangeSlider = ({ value, minPrice, maxPrice, rangeSlided, saveButtonHandle, deleteButtonHandle }) => {
+    const [price, setPrice] = value;
+
+    const handleChange = (event, newValue) => {
+        const [newValueStart, newValueEnd] = newValue;
+        if (newValueEnd - newValueStart <= 5000) setPrice(price);
```

Comment:
```suggestion
        if (newValueEnd - newValueStart <= maxPrice) setPrice(price);
```

maxPrice의 역할로 추정되는데 맞을까요?